substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrManager.java

```diff
@@ -85,13 +83,8 @@ private static void parseFlightRecorderLogging(String option) {
     }
 
     private static void periodicEventSetup() throws SecurityException {
-        FlightRecorder.addPeriodicEvent(InitialSystemProperty.class, InitialSystemProperty::emitSystemProperties);
-        FlightRecorder.addPeriodicEvent(InitialEnvironmentVariable.class, InitialEnvironmentVariable::emitEnvironmentVariables);
-        FlightRecorder.addPeriodicEvent(JVMInformation.class, JVMInformation::emitJVMInformation);
-        FlightRecorder.addPeriodicEvent(OSInformation.class, OSInformation::emitOSInformation);
-        FlightRecorder.addPeriodicEvent(PhysicalMemory.class, PhysicalMemory::emitPhysicalMemory);
-        FlightRecorder.addPeriodicEvent(JavaThreadStatistics.class, JavaThreadStatistics::emitJavaThreadStats);
-        FlightRecorder.addPeriodicEvent(ClassLoadingStatistics.class, ClassLoadingStatistics::emitClassLoadingStats);
+        FlightRecorder.addPeriodicEvent(EveryChunkPeriodEvents.class, EveryChunkPeriodEvents::emitEveryChunkPeriodEvents);
```

Comment:
This change deserves a comment. Also, `emitEveryChunkPeriodEvents` and `emitEndChunkPeriodEvents` could have a better name. I don't understand what they do by reading their name.

Reply:
Done.